mev-boost-rs/src/service.rs

```diff
@@ -69,7 +69,7 @@ impl Service {
         let context =
             if let Some(context) = context { context } else { Context::try_from(&network)? };
         let relays = relays.into_iter().map(Relay::from);
-        let clock = context.clock(None);
+        let clock = context.clock().unwrap();
```

Comment:
we shouldn't just blindly `unwrap` things unless we know they won't panic

here's how I'm thinking about this right now:

https://github.com/ralexstokes/mev-rs/pull/124#discussion_r1319987479